app/views/places/leaderboard.html.erb

```diff
@@ -12,6 +12,7 @@
         <ol>
           <% @leaderboard.top_five.each do |rated_place| %>
             <li>
+              <div class="badge">&nbsp;</div>
```

Comment:
Why is this `&nbsp;` hack needed?

Reply:
Good point. I was tinkering with it to fix the mobile problem and we may be able to remove it now!

Reply:
Nope, I tried removing it and the badge disappears. There might be a nicer way to fix this because on mobile the `div class="badge"` gets the following applied to it, which I think is the problem:

```
.badge:empty {
  display: none;
}
```

I'm out of time for now so happy for someone else to pick this up!

Reply:
I think this is fine assuming removing `.badge:empty` would have other side effects.